var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -597,7 +597,7 @@ def die_without_fortran(self):
                 'OpenMPI requires both C and Fortran compilers!'
             )
 
-    @when('@develop')
+    @when('@master')
```

Comment:
@hppritcha For my understanding, why does this shift from `@develop` to `@master`?

Reply:
because of line 36.  If we replaced the first ```master``` in that line with develop then we would not need to change line 600.  I think i added line 36 with ```develop``` but someone changed it.  I myself don't have a preference, but I do want it to work.  Given all the changes between 4.1.x and 5.0.x/master I was wanting to be able to regularly test our main development branch with spack.